specification/baremetalinfrastructure/resource-manager/Microsoft.BareMetalInfrastructure/preview/2020-08-06-preview/baremetalinfrastructure.json

```diff
@@ -0,0 +1,890 @@
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.BareMetalInfrastructure/bareMetalInstances/{azureBareMetalInstanceName}": {
+      "get": {
+        "tags": [
+          "BareMetalInfrastructure"
+        ],
+        "operationId": "AzureBareMetalInstances_Get",
+        "summary": "Gets an Azure BareMetal instance.",
+        "description": "Gets an Azure BareMetal instance for the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Get an AzureBareMetal instance": {
+            "$ref": "./examples/AzureBareMetalInstances_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureBareMetalInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AzureBareMetalInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "BareMetalInfrastructure"
+        ],
+        "operationId": "AzureBareMetalInstances_Delete",
+        "x-ms-long-running-operation": true,
+        "summary": "Deletes a Azure BareMetal instance.",
+        "description": "Deletes a Azure BareMetal instance with the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Delete an AzureBareMetal instance": {
+            "$ref": "./examples/AzureBareMetalInstances_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureBareMetalInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted AzureBareMetalInstance"
+          },
+          "202": {
+            "description": "Deleting AzureBareMetalInstance"
+          },
+          "204": {
+            "description": "No AzureBareMetalInstance to delete"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
```

Comment:
>patch [](start = 7, length = 5)

Why you don't have PUT call for this?

Reply:
no put call because the hardware is physical and needs to be physically allocated. In the back end, the resource information is populated. In the future there will be one as the CRUD pipeline gets filled out